src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.data.person;
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class Block {
+    public static final String EXAMPLE = "123c";
```

Comment:
Since Address is having all properties (block, street, unit, postal code), there is no need to have these domain classes public. Think about it.